src/Bridges/lazybridgeoptimizer.jl

```diff
@@ -105,5 +105,9 @@ function supports_bridging_constraint(b::LazyBridgeOptimizer, F::Type{<:MOI.Abst
 end
 function bridge_type(b::LazyBridgeOptimizer{BT}, F::Type{<:MOI.AbstractFunction}, S::Type{<:MOI.AbstractSet}) where BT
     update_constraint!(b, F, S)
-    b.best[(F, S)]
+    BridgeType = get(b.best, (F, S), nothing)
```

Comment:
Hmm. There's a naming inconsistency. Local variables should be named like `bridge_type`, not BridgeType.